File: Data Structures & Algorithms/is-anagram/submission-6.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sf = {}
        for i in s:
            if i in sf:
                sf[i] += 1
            else:
                sf[i] = 1
        tf = {}
        for i in t:
            if i in tf:
                tf[i] += 1
            else:
                tf[i] = 1
        return sf == tf

File: Data Structures & Algorithms/sort-an-array/submission-6.cpp
class Solution {
public:
    vector<int> sortArray(vector<int>& nums) {
        int n = nums.size();
        for (int i = 1 ; i < n ; i ++){
            int ele = nums[i], j = i - 1;
            while (j >= 0 && nums[j] > ele){
                nums[j + 1] = nums[j];
                j--;
            }
            nums[j + 1] = ele;
        }
        return nums;
    }
};